### DISCUSSION TITLE
Commissions using Affiliate API

### DISCUSSION THREAD

**Discussion by timothy-kairos** [cloudbet/docs#70] [General]
Hi,  I would like to know if I can gain commissions when the players place bets from my website directly via the affiliate API. Or all the new players must sign up using my affiliate referral link?

**Comment by kgravenreuth** [comment]
Hi @timothy-kairos ,

- If your players sign up via your affiliate referral link, then you can earn commissions on bets placed by them directly on Cloudbet. For more details, you can refer to our [Affiliates FAQ](https://www.cloudbet.com/en/affiliates/faq) section or contact our [Affiliate Support](mailto:affiliate-support@cloudbet.com) .
- Another option is to use the [Cloudbet Trading B2B API](https://www.cloudbet.com/api/?urls.primaryName=Trading%20B2B) which allows you to set your own odds based on the data we supply, enabling you to carve out a profitable margin for each transaction. Alongside margins, you can also take control of and share liability from the individual bets that players or customers place through your platform. To know more, please contact our [dedicated B2B API Team](mailto:b2b@cloudbet.com) .

Let us know if you have any more questions, thanks!

Cheers,
Klaus Gravenreuth
Product & Engineering @ Cloudbet

**Comment by timothy-kairos** [comment]
Dear Klaus,

Great! Thank you for your information!

Best,
Timothy

「Klaus Gravenreuth ***@***.***>」在 2023年10月24日 週二，下午4:48 寫道：
